### ISSUE TITLE
Debug -> Add onfiguration results in "Cannot read property 'resource' of null"

### ISSUE THREAD

**Issue by AndrewPardoe** [Microsoft/vscode#27008]
- VSCode Version: Code 1.12.2 (19222cdc84ce72202478ba1cec5cb557b71163de, 2017-05-10T13:20:36.315Z)
- OS Version: Windows_NT ia32 10.0.16200
- Extensions: none
---

Steps to Reproduce:

1. Remove VS Code from system using ARP
2. Install VS Code from https://code.visualstudio.com/Download
3. Run VS Code
4. Click "Debug"
5. Click "Add Configuration"

Results in:

Error: Cannot read property 'resource' of null.

**Comment by AndrewPardoe** [comment]
I'm in building 25 if you want to see the machine.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/25466
@AndrewPardoe I am in Zurich ;)
Please use the latest vscode 1.13.1 it is fixed there. Also before filing issues it is always good to search for duplicates and check if it works in the latest vscode insiders https://code.visualstudio.com/insiders
Thanks!

**Comment by AndrewPardoe** [comment]
@isidorn: thanks! I actually did check for duplicates but thank you for the reminder. I've realized why I didn't find the issue: GitHub's default issue search looks for "is:open", e.g., https://github.com/Microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20'resource'%20property. I failed to remove that when I searched for duplicates. But note that [your repo's CONTRIBUTING guidance](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md) explicitly says to search **open** issues. 

As for Building 25, I know there are a number of VS Code people across the street. I'm by no means a VS Code expert but I was surprised that something I'd consider a basic functionality could regress in a release. 

Also, I didn't now there was a Preview channel. That's why my repro started with a fresh install. You might consider mentioning that on the CONTRIBUTING page as well. 

I'll be more careful about entering VS Code bugs in the future.
